README.md

```diff
@@ -158,7 +158,11 @@ We welcome contributions, and are especially interested in welcoming [first time
 | `blurredLocation.getPrecision()` | Returns precision of degrees -- represented by width or height of one grid cell. Returns an integer which represents the number of decimal places occupied by one cell. For instance, a precision of 1 will mean 0.1 degrees per cell, 2 will mean 0.01 degrees, and so on | `location.getPrecision()` This would return the precision of the map at the current zoom level. |
 | `blurredLocation.getPlacenameFromCoordinates()` | Returns human-readable location name of a specific latitude and longitude. This would take in 3 arguments namely latitude, longitude and a callback function which is called on success and would return address of the location pinpointed by those co-ordinates| `blurredLocation.getPlacenameFromCoordinates(43, 43, function(result) { console.log(result) }` This would return the output to the console |
 | `blurredLocation.map` | Used to access the Leaflet object | |
-| `blurredLocation.setZoomByPrecision()` | Zooms map to the given precision. | `blurredLocation.setZoomByPrecision(2)` This would zoom the map so that the precision of map becomes 2, and each grid square is `0.01` degrees wide and tall. |
+| `blurredLocation.setZoomByPrecision(precision)` | Zooms map to the given precision. | `blurredLocation.setZoomByPrecision(2)` This would zoom the map so that the precision of map becomes 2, and each grid square is `0.01` degrees wide and tall. |
```

Comment:
`blurredLocation.setZoomByPrecision(precision)`
We can elaborate more on this function to explain how `options.precisionTable` is used to map precision to zoom and this` options.precisionTable` can be taken as input from user.

`Zooms map to the given precision. ` ->  Zooms map according to mapped value of zoom corresponding to passed precision. Then an example explaining this. 
Thanks!

Reply:
Ok!